src/main/scala/nl.knaw.dans.easy.deposit/CommandLineOptions.scala

```diff
@@ -40,12 +45,32 @@ class CommandLineOptions(args: Array[String], configuration: Configuration) exte
        |Options:
        |""".stripMargin)
 
-  val runService = new Subcommand("run-service") {
+  private implicit val stateLabelParser: ValueConverter[State] = singleArgConverter[State](State.withName)
+  private implicit val uuidParser: ValueConverter[UUID] = singleArgConverter[UUID](UUID.fromString)
+
+  val runService: Subcommand = new Subcommand("run-service") {
     descr(
       "Starts EASY Deposit Api as a daemon that services HTTP requests")
     footer(SUBCOMMAND_SEPARATOR)
   }
+  val changeState = new Subcommand("change-state") {
+    descr(
+      "Changes the state of a deposit, when changing to SUBMITTED just the state is changed, the rest of the submit-cycle is not started")
+    val doUpdate: ScallopOption[Boolean] = opt(name = "doUpdate", noshort = true, required = false,
+      descr = s"without this argument only the current status is shown in json format")
+    val state: ScallopOption[State] = opt(name = "label", short = 'l', required = true,
+      descr = s"The label of the new state, one of: ${State.values.mkString(", ")}")
+    val description: ScallopOption[String] = opt(name = "description", short = 'd', required = true,
+      descr = "A desription of the new state")
+    val draftOwnerId: ScallopOption[String] = trailArg(name = "draftOwnerId", required = true,
```

Comment:
`trailArg` has `required = true` as default. no need to state it explicitly.
same below

Reply:
now you are inconsistent with wanting an explicit default for the `opt[Boolean]`